src/main/java/it/gov/pagopa/xpay/dto/XPayErrorEnum.java

```diff
@@ -0,0 +1,54 @@
+package it.gov.pagopa.xpay.dto;
+
+import org.springframework.http.HttpStatus;
+
+public enum XPayErrorEnum {
```

Comment:
@umberto-russo-nexi You can replace the constructor and getter methods with lombok annotations 
```
@AllArgsConstructor
@Getter
```